src/pal/inc/pal.h

```diff
@@ -4679,7 +4679,7 @@ enum {
 //
 typedef struct _RUNTIME_FUNCTION {
     DWORD BeginAddress;
-#ifdef _AMD64_
+#if RUNTIME_FUNCTION_HAS_ENDADDRESS
```

Comment:
Based on what @jkotas said, this is really AMD64 specific and we don't ever want to use it for anything else. That's why I would really prefer keeping it as it was to not to make false expectations that it might be enabled for something else than AMD64.